### PR TITLE
add zh /docs/reference/glossary/cgroup.md

### PR DESCRIPTION
**Pull request by lichuqiang**
to address https://github.com/k8smeetup/website-tasks/issues/3194

### CODE REVIEW

File: content/zh/docs/reference/glossary/cgroup.md [hunk 1]
```diff
@@ -0,0 +1,40 @@
...
+title: cgroup (控制组)
+id: cgroup
+date: 2019-06-25
+full_link:
+short_description: >
+  一组具有可选资源隔离、审计和限制的 Linux 进程。
+
+aka:
+tags:
+- 基础的
```
Comment (tengqm) [inline content/zh/docs/reference/glossary/cgroup.md]: don't translate this tag.

File: content/zh/docs/reference/glossary/cgroup.md [hunk 2]
```diff
@@ -0,0 +1,40 @@
...
+-->
+一组具有可选资源隔离、审计和限制的 Linux 进程。
+
+<!--more--> 
+
+<!--
+cgroup is a Linux kernel feature that limits, accounts for, and isolates the resource usage (CPU, memory, disk I/O, network) for a collection of processes.
+-->
+
+Cgroup 是一个 Linux 内核特性，他对一组进程的资源使用（CPU、内存、磁盘 I/O 和网络等）进行限制、审计和隔离。
```
Comment (tengqm) [inline content/zh/docs/reference/glossary/cgroup.md]:
```suggestion
Cgroup 是一个 Linux 内核特性，对一组进程的资源使用（CPU、内存、磁盘 I/O 和网络等）进行限制、审计和隔离。
```

File: content/zh/docs/reference/glossary/cgroup.md (hunk 1 above)
Comment (lichuqiang) [reply content/zh/docs/reference/glossary/cgroup.md]: done

File: content/zh/docs/reference/glossary/cgroup.md (hunk 2 above)
Comment (lichuqiang) [reply content/zh/docs/reference/glossary/cgroup.md]: done

**Pull request merged**